tools/bazar/controllers/ApiController.php

```diff
@@ -173,6 +174,65 @@ public function createSemanticEntry($formId)
         ]);
     }
 
+    /**
+     * @Route("/api/bazar-list-data", methods={"GET"}, options={"acl":{"public"}})
```

Comment:
```suggestion
     * @Route("/api/entries/bazarlist", methods={"GET"}, options={"acl":{"public"}},priority=2)
```

Je ne suis pas partisan de cette route et je ferais plutôt la proposition que je donne.
La priorité permet d'être sûr de ne pas passer par l'autre méthode

Reply:
`Serait-ce possible d'ajouter la route dans le documentation ? merci (voire méthode du bas)